packages/menu/test/mwc-menu.test.ts

```diff
@@ -299,7 +299,7 @@ suite('mwc-menu', () => {
 
     setup(async () => {
       originalSetTimeout = window.setTimeout;
-      (window as any).setTimeout = (fn: Function) => {
+      (window as any).setTimeout = (fn: () => unknown) => {
```

Comment:
another suggestion:

```ts
const timeoutShim: typeof window.setTimeout = (fn) => {
  (fn as ()=>unknown)();
  ...
  return -1
};
  
window.timeoutShim = a;
```

Reply:
yup i would prefer that too, much nicer to match the type. ill make the change

Reply:
in actual fact this is proving to be difficult to do... as we have `@types/node` which is conflicting. somehow `setTimeout` is node's rather than the browser's.

is there no way we can not have the node types?